libs/feature/editor/src/lib/components/contact-card/contact-card.component.html

```diff
@@ -0,0 +1,22 @@
+<ng-container *ngIf="organization && contact">
+  <div
+    class="flex flex-row border border-gray-200 bg-gray-50 rounded-xl p-4 gap-4"
+  >
+    <img [src]="organization.logoUrl" class="w-[56px] h-[56px] rounded-[4px]" />
+    <div class="flex flex-col w-full">
+      <div class="flex flex-row justify-between">
+        <span class="flex flex-wrap font-bold w-full"
+          >{{ contact.firstName }} {{ contact.lastName }}</span
+        >
+        <gn-ui-button
+          data-test="removeContactButton"
+          type="light"
+          extraClass="w-[20px] h-[20px] flex items-center justify-center"
+          (buttonClick)="removeContact(contact)"
+          ><span class="text-[14px] leading-[0] font-bold">&#x2715;</span>
```

Comment:
```suggestion
          ><span class="material-symbols-outlined">
close
</span>
```
Should we rather use the material symbol? it is more readable at least.